godtools/Views/BaseTractView.swift

```diff
@@ -14,19 +14,34 @@ class BaseTractView: UIView {
     
     var data: XMLIndexer?
     var colors: TractColors?
-    var configurations = TractConfigurations()
+    var configurations: TractConfigurations?
```

Comment:
Thanks for checking this! these properties were from a previous version, now is not needed